onnxruntime/python/tools/tensorrt/perf/build/ort_build_latest.py

```diff
@@ -19,30 +18,27 @@ def parse_arguments():
 def archive_wheel_file(save_path, ort_wheel_file):
     if not os.path.exists(save_path):
         os.mkdir(save_path)
-    p1 = subprocess.Popen(["cp", ort_wheel_file, save_path])
-    p1.wait()
+    subprocess.run(["cp", ort_wheel_file, save_path], check=True)
 
 def install_new_ort_wheel(ort_master_path):
     ort_wheel_path = os.path.join(ort_master_path, "build", "Linux", "Release", "dist") 
-    p1 = subprocess.Popen(["find", ort_wheel_path, "-name", "*.whl"], stdout=subprocess.PIPE)
-    stdout, sterr = p1.communicate()
-    stdout = stdout.decode("utf-8").strip()
+    p1 = subprocess.run(["find", ort_wheel_path, "-name", "*.whl"], stdout=subprocess.PIPE, check=True)
+    stdout = p1.stdout.decode("utf-8").strip()
     ort_wheel = stdout.split("\n")[0]
-    p1 = subprocess.Popen(["pip3", "install", "-I", ort_wheel])
-    p1.wait()
+    subprocess.run(["python", "-m", "pip", "install", "--force-reinstall", ort_wheel], check=True)
     return ort_wheel
 
 def main():
     args = parse_arguments()
 
-    cmake_tar = "cmake-3.17.4-Linux-x86_64.tar.gz" 
+    cmake_tar = "cmake-3.18.4-Linux-x86_64.tar.gz" 
     if not os.path.exists(cmake_tar):
-        p = subprocess.run(["wget", "-c", "https://cmake.org/files/v3.17/" + cmake_tar], check=True)
+        p = subprocess.run(["sudo", "wget", "-c", "https://cmake.org/files/v3.18/" + cmake_tar], check=True)
     tar = tarfile.open(cmake_tar)
     tar.extractall()
     tar.close()
     
-    os.environ["PATH"] = os.path.join(os.path.abspath("cmake-3.17.4-Linux-x86_64"), "bin") + ":" + os.environ["PATH"]
+    os.environ["PATH"] = os.path.join(os.path.abspath("cmake-3.18.4-Linux-x86_64"), "bin") + ":" + os.environ["PATH"]
```

Comment:
FYI, in case you need to update to a newer version of cmake in the future. e.g. cmake 3.20
after extracting the tarfile, the name of the directory changed slightly in the later versions.
linux is lowercase instead of uppercase Linux. so when you add the directory to PATH, you need to make sure it's lower case. e.g. see https://github.com/microsoft/onnxruntime/pull/8055